test/deprecated.jl

```diff
@@ -1,8 +1,9 @@
 using DataFrames, FileIO, RDatasets, Compat
-fn = tempname()
-open(fn, "w") do f
-    redirect_stderr(f) do
-        dataset("datasets", "iris")
+mktemp() do fn
```

Comment:
Looks like the syntax or something is off, maybe? This line is causing failures on all versions and platforms.